cmd/ocsp-responder/main.go

```diff
@@ -27,6 +27,23 @@ import (
 	"github.com/letsencrypt/boulder/sa"
 )
 
+type statsShim struct {
```

Comment:
Can you add a comment to this type that indicates what interface it's implementing/its purpose?